src/web/middleware/sentry.rs

```diff
@@ -72,9 +103,41 @@ where
 
         Box::pin(self.service.call(sreq).and_then(move |sresp| {
             // handed an actix_error::error::Error;
-            // Fetch out the tags (in case any have been added.)
+            // Fetch out the tags (in case any have been added.) NOTE: request extensions
+            // are NOT automatically passed to responses. You need to check both.
+            if let Some(t) = sresp.request().extensions().get::<Tags>() {
+                debug!("Found request tags: {:?}", &t.tags);
+                for (k, v) in t.tags.clone() {
+                    tags.tags.insert(k, v);
+                }
+            };
+            if let Some(t) = sresp.response().extensions().get::<Tags>() {
+                debug!("Found response tags: {:?}", &t.tags);
+                for (k, v) in t.tags.clone() {
+                    tags.tags.insert(k, v);
+                }
+            };
+            // add the uri.path (which can cause influx to puke)
+            tags.extra.insert("uri.path".to_owned(), uri);
             match sresp.response().error() {
-                None => {}
+                None => {
+                    // Middleware errors are eaten by current versions of Actix. Errors are now added
+                    // to the extensions. Need to check both for any errors and report them.
+                    if let Some(events) = sresp.request().extensions().get::<Vec<Event<'static>>>()
```

Comment:
we could `remove()` instead of `get()` here to avoid the clones, but Event looks probably cheap anyway